backends/graphics/surfacesdl/surfacesdl-graphics.h

```diff
@@ -339,9 +342,12 @@ class SurfaceSdlGraphicsManager : public SdlGraphicsManager {
 	};
 
 	// Dirty rect management
-	SDL_Rect _dirtyRectList[NUM_DIRTY_RECT];
+	SDL_Rect _dirtyRectList[2 * NUM_DIRTY_RECT];
```

Comment:
why multiple by 2?

Reply:
Because with double buffering we need to redraw what was updated in current frame but also in previous frame, hence doubling

Reply:
Done